app/src/main/java/com/huji/foodtricks/buddies/EventModel.java

```diff
@@ -57,29 +58,31 @@ public Date getTime() {
         return time;
     }
 
-    public String getEventType() {
-        return eventType;
+    public void setOrganizerID(String newID) {
+        this.organizerID = newID;
     }
 
-    public void setEventType(String eventType) {
-        this.eventType = eventType;
+    public List<String> getInviteesIDs() {
+        return new ArrayList<>(inviteesIDs);
     }
-
-    public int getAttendingCount(){
+    public int attendingCount(){
         return attendanceProvider.getAttendingCount();
     }
 
-    public int getNotAttendingCount(){
+    public int notAttendingCount(){
         return attendanceProvider.getNotAttendingCount();
     }
-    public int getTentativesCount(){
+    public int tentativesCount(){
         return attendanceProvider.getTentativesCount();
     }
 
-    public int getNonResponsiveCount(){
+    public int nonResponsiveCount(){
         return attendanceProvider.getNonResponsiveCount();
     }
 
+    public boolean isUserOrganizer(String userID) {
```

Comment:
A function that returns the organizer's ID might be more useful (for sending notification for example)